sdcm/utils/decorators.py

```diff
@@ -321,3 +321,15 @@ def wrapper(*args, **kwargs):
                 raise UnsupportedNemesis("Capacity Issue") from ex
             raise
     return wrapper
+
+
+def optional_stage(stage_name):
```

Comment:
A docstring explaining the usage and  pointing to docs explain this one, could be useful

Reply:
Added doc.string